modules/cli/controllermanifest/template.go

```diff
@@ -63,7 +63,7 @@ func objects(namespace, controllerImage string, cfg string, crdsOnly bool) ([]ru
 	objs := []runtime.Object{
 		basic.Namespace(namespace),
 		serviceAccount,
-		basic.Deployment(namespace, genericName, controllerImage, "", serviceAccount.Name),
+		basic.Deployment(namespace, genericName, controllerImage, "", serviceAccount.Name, true),
```

Comment:
I don't think your will need this though.. The method is not used anymore

Reply:
The method is not used for the manager image anymore?